timesketch/api/v1/resources/sketch.py

```diff
@@ -341,7 +341,9 @@ def get(self, sketch_id):
                 x for x, y in analyzer_manager.AnalysisManager.get_analyzers()
             ],
             mappings=list(mappings),
-            stats=stats_per_index
+            stats=stats_per_index,
+            labels=self.datastore.get_sketch_labels(sketch.id, sketch_indices),
+            sketch_labels=[label.label for label in sketch.labels]
```

Comment:
`sketch.labels` is just the labels themselves, we already have the `label_string` with the same information, is this needed?

Reply:
label_string() gives me a json encoded string of the list of labels. I opted to do this instead of deserialize. I'm jsonifying this so I need a dict.

Reply:
I did however rename it to filter_labels to make it clearer what it is.